ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/dorhook.js

```diff
@@ -89,12 +89,11 @@
             return false;
     }
 
-    function getGuideContainerProperties() {
-        var formPath = getFormPath(ns.ContentFrame.getContentPath());
+    function getGuideContainerProperties(contentFramePath) {
```

Comment:
minor: name of argument is incorrect. 
contentFrame has no path.
it is the path of the editable. we can rename it as `editablePath`

Reply:
done